jax_galsim/image.py

```diff
@@ -1345,6 +1386,6 @@ def Image_or(self, other):
 Image.__rand__ = Image_and
 Image.__rxor__ = Image_xor
 Image.__ror__ = Image_or
-Image.__iand__ = Image_and
-Image.__ixor__ = Image_xor
-Image.__ior__ = Image_or
+Image.__iand__ = Image_iand
+Image.__ixor__ = Image_ixor
+Image.__ior__ = Image_ior
```

Comment:
Why do this indirection?  Why not just define them in the Image class directly?

cf. https://github.com/GalSim-developers/GalSim/blob/releases/2.4/galsim/image.py#L1612

Reply:
Hmm, I don't remember why we did it this way instead of the Galsim way. @EiffL @jecampagne do you remember?